server/controllers/ArticlesController.js

```diff
@@ -88,13 +88,20 @@ class ArticlesController {
     )
       .then((article) => {
         if (!article) {
-          return res.status(400).json({
+          res.status(400).json({
```

Comment:
Why did you remove the return statement?
Stop writing mad code.

Reply:
Thanks for the feedback Stephen, I will do better next time